app/src/main/java/chat/rocket/android/app/DrawableHelper.kt

```diff
@@ -37,7 +44,7 @@ object DrawableHelper {
      * @see wrapDrawables
      * @see tintDrawable
      */
-    fun wrapDrawable(drawable: Drawable) = DrawableCompat.wrap(drawable)
+    fun wrapDrawable(drawable: Drawable): Drawable = DrawableCompat.wrap(drawable)
```

Comment:
I think most of our Helper classes could be extension functions
```
fun Drawable.wrap() = DrawableCompat.wrap(this)
fun Context.drawableForId(id: Int) = this.resources.getDrawable(id, null)
fun Context.getColorCompat(resId: Int) = ContextCompat.getColor(this, resId)
fun Array<Drawable>.tint(context: Context, resId: Int) {
    for (drawable in this) {
        DrawableCompat.setTint(drawable, context.getColorCompat(resId))
    }
}
```

Just some samples...

Reply:
It is a good case to be an extension function. I'll add a TODO. Thanks!